packages/start/src/runtime/server-handler.ts

```diff
@@ -128,15 +128,17 @@ async function handleServerFunction(h3Event: HTTPEvent) {
     // This should never be the case in "proper" Nitro presets since node.req has to be IncomingMessage,
     // But the new azure-functions preset for some reason uses a ReadableStream in node.req (#1521)
     const isReadableStream = h3Request instanceof ReadableStream;
-    const isH3EventBodyStreamLocked = isReadableStream && h3Request.locked;
+    const hasReadableStream = h3Request.body instanceof ReadableStream;
+    const isH3EventBodyStreamLocked =
+      (isReadableStream && h3Request.locked) || (hasReadableStream && h3Request.body.locked);
```

Comment:
in other words, this was handling a case where `h3Request` is a locked `ReadableStream` but in the netlify case `h3Request.body` was a locked `ReadableStream... so I just added handling for that too

Reply:
Nice seems simple and not that impacting. Thank you for taking the time to look at this.